cli/foundation-models/system/inference/automatic-speech-recognition/asr-batch-endpoint.sh

```diff
@@ -89,8 +89,8 @@ az ml batch-endpoint delete --name $endpoint_name $workspace_info --yes || {
     echo "endpoint delete failed"; exit 1;
 }
 
-# 6. Delete the compute cluster
-az ml compute delete --name cpu-cluster $workspace_info --yes || {
-    echo "compute delete failed"; exit 1;
-}
+# 6. Delete the compute cluster (Comment out the below lines to delete the created cluster)
```

Comment:
# 6. Delete the compute cluster (Uncomment the below lines to delete the created cluster)